### PR TITLE
Lm show h2console link

### PR DESCRIPTION
**Pull request by miguel-luis9**
FEATURE: H2 Console option on navbar #29

Depending on SHOW_H2_CONSOLE_ENABLED, it will show H2 Console

### CODE REVIEW

**Review by pconrad — CHANGES_REQUESTED** [review]
Please remove commented out lines.
See: https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html

File: src/main/resources/application-production.properties [hunk 1]
```diff
@@ -3,3 +3,6 @@ spring.datasource.username=${JDBC_DATABASE_USERNAME}
 spring.datasource.password=${JDBC_DATABASE_PASSWORD}
 
 spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQL9Dialect
+app.showSwaggerUILink=${SHOW_SWAGGER_UI_LINK:${env.SHOW_SWAGGER_UI_LINK:false}}
+# app.showH2ConsoleEnabled=${SHOW_H2_CONSOLE_ENABLED:${env.SHOW_H2_CONSOLE_ENABLED:false}}
```
[the remark this answers is not in the document]
Comment (miguel-luis9) [reply src/main/resources/application-production.properties]: I apologize for the late reply but I took out the commented lines.

**Review by ericknee — DISMISSED** [review]
LGTM - no more comments in code

**Review by pconrad — APPROVED** [review]
LGTM

**Pull request merged**